SeleniumWebDriverBasics/SeleniumWebDriverBasics/Tests/YandexUnitTests.cs

```diff
@@ -0,0 +1,239 @@
+﻿using NUnit.Framework.Internal;
+using SeleniumWebDriverBasics.Tests.Base;
+using SeleniumWebDriverBasics.Utilities;
+using SeleniumWebDriverBasics.WebObjects.Pages;
+
+namespace SeleniumWebDriverBasics.Tests
+{
+    [TestFixture]
+    public class YandexUnitTests : BaseTest
+    {
+        private readonly LoginPage loginPage = new LoginPage();
+        private readonly EmailPage emailPage = new EmailPage();
+
+        [Test]
+        public void GetLoggedInToYandex()
+        {
+            //Act
```

Comment:
A message about which test case is running can be added at the beginning. For example, "'TestName' execution is started".